ProjFS.Mac/PrjFSKext/PrjFSProviderUserClient.cpp

```diff
@@ -29,7 +29,7 @@ const IOExternalMethodDispatch PrjFSProviderUserClient::ProviderUserClientDispat
         {
             .function =                 &PrjFSProviderUserClient::kernelMessageResponse,
             .checkScalarInputCount =    2, // message id, response type
-            .checkStructureInputSize =  0,
+            .checkStructureInputSize =  kIOUCVariableStructureSize, // Arbitrary data buffer, contents depends on kernel request
```

Comment:
If this change isn't needed for this PR can we please put it up in a separate PR?  I don't mind small PR for any changes, just want to keep this PR to task at hand if possible.

Reply:
I was expecting #910 to go in before this PR, so this would have disappeared in a rebase. I've removed it now and fixed up the merge conflicts, as #910 is bogged down in other merge conflicts and I don't have the bandwidth to keep both that PR and this one up to date right now. I'll revisit #910 as soon as this one is merged.